docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,65 @@
 # Control of documents
 
-Bioinformaticians work in a git environment. Git is a distributed version control system that tracks changes in any set of computer files, usually used for coordination work among programmers collaboratively developing source code during software development. Software procedures and related documentation needs to be controlled, and according to ISO-15189, this can be in any form or type of medium. For software development, there are several reasons to not include all the software documentation within the medical laboratories chosen quality management system.
+## Introduction 
 
-- Software development is done in a distributed version control system. These systems are used for coordination work among programmers. Often these systems also allow for documentation of procedures. With the following advantages a. Automatically link to i.e. code, issues, releases, and release notes. b. Possibility to include code blocks that allow for copy-paste into the console/terminal. It’s fast and readable for bioinformaticians. For example, code blocks can be used for general procedures to start a pipeline, or for trouble shooting.
```

Comment:
Add this back -> more arguments to explain why we need a different a different QMS

Reply:
Insist code blocks have no errors on this kind of format compared to copy pasted from a word document

Reply:
Referring to issues etc in our documentation.
Mention that when doing a software update, we must have the corresponding SOP in our docs be updated as well. It might be possible to do automatically.